zebra-network/src/peer_set/set/tests/prop.rs

```diff
@@ -90,17 +96,85 @@ proptest! {
             Ok::<_, TestCaseError>(())
         })?;
     }
+
+    /// Test if requests are broadcasted to the right number of peers.
+    #[test]
+    fn broadcast_to_peers(
+        peer_versions in any::<PeerVersions>(),
+    ) {
+        // Get a dummy block hash to help us construct a valid request to be broadcasted
+        let block: block::Block = zebra_test::vectors::BLOCK_MAINNET_10_BYTES
+            .zcash_deserialize_into()
+            .unwrap();
+        let block_hash = block::Hash::from(&block);
+
+        // Get the current valid peer version
+        let current_mainnet_version = Version::min_specified_for_upgrade(
+            Network::Mainnet,
+            NetworkUpgrade::Canopy
+        );
```

Comment:
This part of the test seems fragile, it will break as soon as we update the minimum peer version after NU5 activation.

Can we use `CURRENT_NETWORK_PROTOCOL_VERSION` instead?